apps/payments/next/app/[offeringId]/error/page.tsx

```diff
@@ -70,96 +31,91 @@ export default async function Error({ params }: { params: CheckoutParams }) {
   const cartData = getCartData(cartId);
   const [contentful, cart] = await Promise.all([contentfulData, cartData]);
   /* eslint-disable @typescript-eslint/no-unused-vars */
-  const cartService = await app.getCartService();
-
-  // Note: commented out as this portion is not part of main tasks
-  // taken from T&P to assist in translations
-  // comments retained from T&P
-  // TODO - Temporary
-  // Identify an approach to ensure we don't have to perform this logic
-  // in every component/page that requires localization.
-  // const languages = headers()
-  //   .get('Accept-Language')
-  //   ?.split(',')
-  //   .map((language) => language.split(';')[0]);
-
-  // TODO
-  // Move to instantiation on start up. Ideally getBundle's, generateBundle, is only called once at startup,
-  // and then that instance is used for all requests.
-  // Approach 1 (Experimental): https://nextjs.org/docs/app/building-your-application/optimizing/instrumentation
-  // Approach 2 (Node global): https://github.com/vercel/next.js/blob/canary/examples/with-knex/knex/index.js#L13
-  // const l10n = await getBundle(languages);
+  // const cartService = await app.getCartService();
+
+  const languages = headers()
+    .get('Accept-Language')
+    ?.split(',')
+    .map((language) => language.split(';')[0]);
+  const l10n = await getBundle(languages);
+
+  const getErrorReason = (reason: string) => {
+    switch (reason) {
+      case 'iap_upgrade_contact_support':
+        return {
+          buttonFtl: 'payment-error-manage-subscription-button',
+          buttonLabel: 'Manage my subscription',
+          message:
+            'You can still get this product — please contact support so we can help you.',
+          messageFtl: 'iap-upgrade-contact-support',
+        };
+      default:
+        return {
+          buttonFtl: 'payment-error-retry-button',
+          buttonLabel: 'Try again',
+          message: 'Something went wrong. Please try again later.',
+          messageFtl: 'basic-error-message',
+        };
+    }
+  };
 
   return (
     <>
-      {/* error page header and subheader */}
-      <header className="subscription-title page-title-container desktop:pt-10">
-        <h1 className="font-semibold leading-8 mb-1 text-grey-600 text-xl">
-          Error confirming subscription…
+      <header className="page-title-container">
+        <h1 className="page-header">
+          {l10n.getMessage('subscription-error-title')?.value?.toString() ||
+            'Error confirming subscription…'}
         </h1>
-        <div className="flex items-center justify-center gap-2 text-green-900 mb-4">
-          <CheckIcon />
-          <h2 className="font-semibold text-sm">30-day money-back guarantee</h2>
```

Comment:
This is currently not a heading... I think if this line is optional, we shouldn't consider it as a heading.

Reply:
Aligned.